components/console/introduction.rst

```diff
@@ -140,6 +140,9 @@ output. For example::
     // white text on a red background
     $output->writeln('<error>foo</error>');
 
+The closing tag can be replaced by just ``</>``. This makes the component to
+stop applying any style defined by the immediately previous open tag.
```

Comment:
I don't like my original text ... but I don't like either the new proposal. My fear is that reader would say: *obviously* the close tag closes the opened tag!!!  In this case, the close tag is not about closing tags but about "neutralizing" the effects applied in the previously opened tag. I don't know if I explained myself because it's not easy to explain what I want to say :(